pkg/pods/pod.go

```diff
@@ -378,7 +378,7 @@ func (pod *Pod) Verify(manifest *Manifest, keyring openpgp.KeyRing) error {
 
 		var launchableDigest map[string]string
 
-		if stanza.DigestSignatureLocation == "" || keyring == nil {
+		if stanza.DigestSignatureLocation == "" || verifier == nil {
```

Comment:
Actually, the default value for a channel is `nil`. [[ref]](https://golang.org/ref/spec#Channel_types)

But I'm not sure whether we want it to be null.  The checks for `nil` throughout the code seem to suggest that the value means: "don't do any signature checks." It's explicitly allowed in [setup.go](https://github.com/square/p2/blob/master/pkg/preparer/setup.go#L120), and disallowed in [p2-preparer/main.go](https://github.com/square/p2/blob/master/bin/p2-preparer/main.go#L26). I guess this is used just for testing?